### PR TITLE
refactor(core+telegram): Improve /balance command performance #123

### PR DESCRIPTION
**Pull request by panapol-p**
### What have you changed and why?
Create new function to get all pair in one time

##### Related Issues
Improve /balance command performance #123

### CODE REVIEW

File: service/service.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ type Feeder interface {
 
 type Broker interface {
 	Account() (model.Account, error)
+	PositionAllPairs(pairs []string) (assets []model.Assets, err error)
```
Comment (rodrigo-brito) [inline service/service.go:25]: I think we don't need this function. We can use the `Account()` which return all pairs balance. Using Account you optimize the result getting all pairs balance once.

**Review by rodrigo-brito — CHANGES_REQUESTED** [review]
It looks good, but I think we can use the current functions provided by the `exchange` object.

File: service/service.go (hunk 1 above)
Comment (panapol-p) [reply service/service.go:25]: thanks for your reviewed, I will revise follow this one.

**Comment by panapol-p** [comment]
supersede by #184

**Pull request closed without merging**